### PR TITLE
Upgrade to neutrino v7, fix inspector task tab switching

### PR DESCRIPTION
**Pull request by eliperelman**

### CODE REVIEW

**Comment by eliperelman** [comment]
Clearing review, not ready yet.

**Comment by eliperelman** [comment]
Switching to Neutrino v7, start using prettier, add husky and lint-staged. Fixing all the new linting problems.

What does this mean?

Your changes are automatically linted and fixed when you commit (for ones that are auto-fixable). 

**No more linting reviews.**

Yes, you heard that right. I am no longer going to do linting gruntwork on my reviews. Whatever eslint and prettier fix and format is what I am going to take.

Feel free to propose rule changes in `.neutrinorc.js` in the future.

**Comment by eliperelman** [comment]
Realistically I am only looking for a review on:

.neutrinorc.js
package.json
views/UnifiedInspector/Inspector.js

**Comment by eliperelman** [comment]
In views/UnifiedInspector/Inspector.js, only review `componentWillReceiveProps`, the rest should be linting fixes.

File: .neutrinorc.js [hunk 1]
```diff
@@ -17,13 +17,27 @@ module.exports = {
   use: [
     ['neutrino-preset-mozilla-rpweb', {
       eslint: {
+        baseConfig: {
+          extends: [
+            'eslint-config-prettier'
+          ]
+        },
+        plugins: ['eslint-plugin-prettier'],
```
Comment (helfi92) [inline .neutrinorc.js]: Fantastic idea!

**Pull request merged**